components/bower_components/system.js/.bower.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
They are included only because when I was deploying I had problems getting django-bower to work on the server. It was a temporary workaround that will be fixed eventually.